packages/engine/src/Actions.js

```diff
@@ -132,12 +131,24 @@ class Actions {
         params: parsedAction.params,
       });
     } catch (error) {
+      responses[action.id] = { error, index, type: action.type };
+      const { output: parsedMessages, errors: parserErrors } = this.context.parser.parse({
+        actions: responses,
+        event,
+        arrayIndices,
+        input: action.messages,
+        location: block.blockId,
+      });
+      if (parserErrors.length > 0) {
+        // this condition is very unlikely since parser errors usually occur in the first parse.
+        throw { error: parserErrors[0], type: action.type, index };
+      }
       closeLoading();
       this.displayMessage({
-        defaultMessage: error.lowdefyMessage || 'Action unsuccessful',
+        defaultMessage: error.message,
```

Comment:
Do we not want to rather show more generic message to users?

Reply:
So the actual error message by default a more standard thing to do, we have `messages.error` if the developer would like to make the response customized. Also, if a user told you the error was .... the chances of finding / fixing it are better. Thoughts?